crates/sui-replay/src/replay.rs

```diff
@@ -153,7 +151,12 @@ pub struct Storage {
 impl std::fmt::Display for Storage {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         writeln!(f, "Live object store")?;
-        for (id, obj) in self.live_objects_store.iter() {
+        for (id, obj) in self
+            .live_objects_store
+            .lock()
+            .expect("Unable to lock")
```

Comment:
You could probably just do unwrap here and for all lock calls since locks only error if they're poisoned.

Reply:
I was just following the same style as in the rest of the crate, but can change it, np

Reply:
ah no worries, it was just a nit